client/gui/view/View.h

```diff
@@ -67,7 +67,10 @@ class View : public IShowActivatable
 public:
 	View(ui16 used=0, Point offset=Point());
 	virtual ~View();
-
+	
+	View *getParent() const;
+	void setParent(View *parent);
```

Comment:
This should be removed, or made protected it you really want to have setter anyway.
You should use `addChild` `removeChild` from outside

Reply:
I will. Give me some time, I just want rework all these things, so I will made some things temporary :).